### ISSUE TITLE
Ver 0.9.3 is not released to npm yet.

### ISSUE THREAD

**Issue by shunjikonishi** [city41/node-sql-fixtures#28]
I want to use showWarning option.  
But v0.9.3 is not released yet.

Regards

Shunji

**Comment by shunjikonishi** [comment]
Now I could use showWarning with 0.9.0.
The reason why I coudn't use it before is caused by document mistake.

In "Additional Options to create()" section, it said

```
sqlFixtures.create(dbConfig, options, dataSpec, function(err, result) {
  ...
});
```

But it is wrong about sequence of arguments.
Correct arguments seq is

```
sqlFixtures.create(dbConfig, dataSpec, options function(err, result) {
  ...
});
```

BTW though I'm using postgres, sql-fixtures shows this warning.
Why?

Regards

Shunji Konishi

**Comment by city41** [comment]
do you have a small repro that shows postgres showing the warning?

`showWarning` is in 0.9.0, and yes that was a documentation mistake. 

Sorry for the late reply, I've been really busy lately. But I will look into this in detail this weekend.

**Comment by shunjikonishi** [comment]
@city41 
Thank you for reply. Your reply is quick :)

My code is here.

https://gist.github.com/shunjikonishi/2db1bde3286b1bc548f2

**Comment by city41** [comment]
Thanks @shunjikonishi 

I fixed the documentation bug, and also published `0.10.0`. It has no real changes in it, just pushing the versions past 0.9 since I made a mess of it.

I am addressing the `showWarning` issue in a separate bug now.

**Comment by shunjikonishi** [comment]
Thanks
